### PR TITLE
added script scaffolding

### PR DESCRIPTION
**Pull request by timwright12**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -62,35 +62,32 @@ node('vetsgov-general-purpose') {
...
+          'nightwatch-e2e': {
+            sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p nightwatch up -d && docker-compose -p nightwatch run --rm --entrypoint=npm -e BABEL_ENV=test -e BUILDTYPE=vagovstaging content-build --no-color run nightwatch:docker"
+          },
+
+          'nightwatch-accessibility': {
+            sh "export IMAGE_TAG=${commonStages.IMAGE_TAG} && docker-compose -p accessibility up -d && docker-compose -p accessibility run --rm --entrypoint=npm -e BABEL_ENV=test -e BUILDTYPE=vagovstaging content-build --no-color run nightwatch:docker -- --env=accessibility"
+          }
+        )
+      } catch (error) {
+        commonStages.slackNotify()
```
Comment (billfienberg) [inline Jenkinsfile:81]: I think we want to leave `slackNotify` commented out.

File: src/platform/site-wide/tests/accessible-modal.e2e.spec.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const E2eHelpers = require('../../testing/e2e/helpers');
```
Comment (billfienberg) [inline src/platform/site-wide/tests/accessible-modal.e2e.spec.js:1]: Looks like this file was copied from `vets-website`. 

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/tests/accessible-modal.e2e.spec.js

**Review by billfienberg — COMMENTED** [review]
Is the `src/site/stages/build/content-validation.js` supposed to be empty?

**Comment by timwright12** [comment]
> Is the `src/site/stages/build/content-validation.js` supposed to be empty?

@billfienberg yeah, it was a placeholder for running the script in circle, you can remove it if you don't end up using it

**Review by billfienberg — APPROVED** [review]

**Pull request merged**